cobra/solvers/coin.py

```diff
@@ -27,7 +27,7 @@ def objectiveValue_(self):
 
 
 def create_problem(cobra_model, objective_sense="maximize", **kwargs):
-    m = cobra_model.to_array_based_model()
+    m = cobra_model.copy().to_array_based_model()
```

Comment:
sorry about that, deleted + re-pushed a commit I thought didn't require that copy, but turns out it didn't work. Using the deepcopy keyword to clean things up a bit now.